### PR TITLE
docs(readme): update readme for openapi

### PR DESCRIPTION
**Pull request by ddysher**
**What this PR does / why we need it**:

docs(readme): update readme for openapi

**Which issue(s) this PR fixes** *(optional, close the issue(s) when PR gets merged)*:

Fixes #

**Special notes for your reviewer**:

/cc @liubog2008 @kdada 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: examples/getting-started/openapi/pkg/api/openapi_generated.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// +build !ignore_autogenerated
```
Comment (kdada) [inline examples/getting-started/openapi/pkg/api/openapi_generated.go]: Can the comment pass `hack/verify_boilerplate.py `?

Comment (ddysher) [reply examples/getting-started/openapi/pkg/api/openapi_generated.go]: Yes

**Pull request merged**